### DISCUSSION TITLE
옵챗 유저의 userId는 어떻게 따오면 될까요?

### DISCUSSION THREAD

**Discussion by williameom5678** [storycraft/node-kakao#347] [Q&A] [answered]
오픈채팅방 유저의 userId와 linkId를 따오는 방법이 궁금합니다.

**Comment by raravel** [comment, marked as answer]
채팅 이벤트를 받았을 땐 다음 함수를 사용해서 전송자 정보를 가져옵니다.

https://github.com/storycraft/node-kakao/blob/b399e23e5ff8068c53dcfb8ba0c07fcd9f6cd1bc/src/talk/chat/index.ts#L117-L126

사용법:
```typescript
client.on('chat', (chat: TalkChatData, channel: TalkOpenChannel) => {
    const sender: ChannelUserInfo = chat.getSenderInfo(channel);
});
```

`TalkOpenChannel` 클래스에서는 다음 함수를 가지고 유저 정보를 가져올 수 있습니다.

https://github.com/storycraft/node-kakao/blob/b399e23e5ff8068c53dcfb8ba0c07fcd9f6cd1bc/src/talk/openlink/talk-open-channel.ts#L129-L135

**Reply by williameom5678** [reply]
넵 감사합니다 ㅎㅎ
